media/static-bundles.json

```diff
@@ -1588,6 +1588,12 @@
         "js/firefox/enterprise/sla.js"
       ],
       "name": "firefox-enterprise-sla"
+    },
+    {
+      "files": [
+        "protocol/js/protocol-newsletter.js"
```

Comment:
There's a JS error when the page loads:

```javascript
TypeError: Argument 1 of Window.getComputedStyle is not an object.
```

As a result, when I submit the form, it doesn't display the custom success message that's in the page. I think this may be because there's no `newsletter-details` container to expand?

Reply:
I'm not sure what content to put in `newsletter-details` for this form. 🤔

Reply:
If you take a look at other newsletter forms, such as on the [home page]([url](https://www.mozilla.org/en-US/)), then inspect the dom you can see what's inside there.

Reply:
I nested information under `newsletter-details` but I'm still struggling to trigger the custom success message. The current success message is from `newsletter/one_newsletter_signup.html`

Reply:
The current success message is already in the page. If the JS is working, it should show the message depending on the class names used in the newsletter JS.

Reply:
Right! I feel a little lost because I'm not receiving JS console errors for some reason.

Reply:
Looks like you need to include `"js/newsletter/form-protocol.js"` as well

Reply:
When testing it out there may be some input ID's you need to update in the HTML to match what's expected in the JS. I think it also expects a value for country, which is something this particular form doesn't have - so you may need to make that an optional thing that gets passed.

Reply:
ah ok! I think I have provided a fix for this & things seem to be working as expected.

```
<input type="hidden" name="lang" id="id_lang"  value="en" aria-required="false">
<input type="hidden" name="country" id="id_country"  aria-required="false">
```